engine/class_modules/sc_death_knight.cpp

```diff
@@ -6143,6 +6176,19 @@ struct virulent_plague_t : public death_knight_spell_t
       base_multiplier *= .375;
     }
   }
+
+  void impact( action_state_t* state ) override
```

Comment:
I got stumped on how to switch all 3 specs to a pattern of putting superstrain in the dots themselves instead of the applicators but this fixes superstrain not working for unholy in aoe and leaves outbreak as a ST spell.  Might be worth revisiting for cleanliness but it gets something working into Taez's hands for now.

Reply:
A solution would be to add a boolean variable to the diseases and create different objects for superstrain usage. But this implementation works fine as is.